modules/tests/src/test/scala/dev/profunktor/redis4cats/TestScenarios.scala

```diff
@@ -592,10 +593,16 @@ trait TestScenarios { self: FunSuite =>
                )
              )
       _ <- IO(assertEquals(list, List("Let", "us", "have", "fun")))
+      boolReadOnly <- redis.evalReadOnly("return true", ScriptOutputType.Boolean, List("Foo"))
+      _ <- IO(assert(boolReadOnly))
       _ <- redis.eval(statusScript, ScriptOutputType.Status, List("test"), List("foo"))
+      either <- redis.evalReadOnly(statusScript, ScriptOutputType.Status, List("test"), List("foo")).attempt
+      _ <- IO(assert(either.left.exists(_.isInstanceOf[RedisReadOnlyException])))
```

Comment:
Considering **Redis** has a specific `READONLY` _error_ that is raised when trying to write to a read-only replica, it was only natural to me to assume trying to write on a read-only script would raise the same _error_. But, apparently, that is not the case.
I will fix this assertion shortly.